pyproject.toml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2014-2024  Barnstormer Softworks, Ltd.
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+[project]
+name = "geni-lib"
+version = "0.9.9.5"
```

Comment:
I took the version from the old setup.py file, but it might not be correct. Usually, semantic versioning only has three numbers.